### DISCUSSION TITLE
Hono Jwt helper exp params

### DISCUSSION THREAD

**Discussion by vikramnagwal** [honojs/hono#3790] [Q&A] [answered]
Hono's JWT helper takes `exp`, `iat`, and `nbf` as parameters along with a `payload` option. The `exp` parameter only accepts numbers. My question is: how should I provide these numbers? Is the default unit seconds? For instance, if I want to set the expiration time to 15 minutes, should I use `900` or `15`? 

For example: `sign({ payload, exp: 900 }, secret)`.

I apologize if I’m missing something.

**Comment by Ozod0107** [comment, marked as answer]
exp: Math.floor(Date.now() / 1000) + 86400 (1 day)

you should use like this

**Reply by bestwishCT** [reply]
// 10 minute
const payload = {
      sub: 'admin',
      role: 'admin',
      exp: Math.floor(Date.now() / 1000) + 60 * 10,
    };

**Comment by vikramnagwal** [comment]
thanks mate @Ozod0107
